tests/issues/issue-2475.php

```diff
@@ -0,0 +1,103 @@
+            'Platform_Version' => '10.0',
+            'Platform_Description' => 'Windows 10',
+            'Platform_Bits' => '64',
+            'Platform_Maker' => 'Microsoft Corporation',
+            'Alpha' => false,
+            'Beta' => false,
+            'Frames' => true,
+            'IFrames' => true,
+            'Tables' => true,
+            'Cookies' => true,
+            'JavaScript' => true,
+            'VBScript' => false,
+            'JavaApplets' => false,
+            'isSyndicationReader' => false,
+            'isFake' => false,
+            'isAnonymized' => false,
+            'isModified' => false,
+            'CssVersion' => '3',
+            'Device_Name' => 'Windows Desktop',
+            'Device_Maker' => 'unknown',
+            'Device_Type' => 'Desktop',
+            'Device_Pointing_Method' => 'mouse',
+            'Device_Code_Name' => 'Windows Desktop',
+            'Device_Brand_Name' => 'unknown',
+            'RenderingEngine_Name' => 'Blink',
+            'RenderingEngine_Version' => 'unknown',
+            'RenderingEngine_Maker' => 'Google Inc',
+        ],
+        'lite' => false,
+        'standard' => true,
+        'full' => true,
+    ],
+    'issue-2370-B' => [
+        'ua' => 'Mozilla/5.0 (iPhone; CPU iPhone OS 12_3_1 like Mac OS X) AppleWebKit/602.1.50 (KHTML, like Gecko) CriOS/99.0.2924.75 Mobile/14E5239e Safari/602.1',
+        'properties' => [
+            'Comment' => 'Chrome 99.0',
+            'Browser' => 'Chrome',
+            'Browser_Type' => 'Browser',
+            'Browser_Bits' => '64',
+            'Browser_Maker' => 'Google Inc',
+            'Browser_Modus' => 'unknown',
+            'Version' => '99.0',
+            'Platform' => 'iOS',
+            'Platform_Version' => '12.3',
+            'Platform_Description' => 'iPod, iPhone & iPad',
+            'Platform_Bits' => '64',
+            'Platform_Maker' => 'Apple Inc',
+            'Alpha' => false,
+            'Beta' => false,
+            'Frames' => true,
+            'IFrames' => true,
+            'Tables' => true,
+            'Cookies' => true,
+            'JavaScript' => true,
+            'VBScript' => false,
+            'JavaApplets' => false,
+            'isSyndicationReader' => false,
+            'isFake' => false,
+            'isAnonymized' => false,
+            'isModified' => false,
+            'CssVersion' => '3',
+            'Device_Name' => 'iPhone',
+            'Device_Maker' => 'Apple Inc',
+            'Device_Type' => 'Mobile Phone',
+            'Device_Pointing_Method' => 'touchscreen',
+            'Device_Code_Name' => 'iPhone',
+            'Device_Brand_Name' => 'Apple',
+            'RenderingEngine_Name' => 'WebKit',
+            'RenderingEngine_Version' => 'unknown',
+            'RenderingEngine_Maker' => 'Apple Inc',
+        ],
+        'lite' => true,
+        'standard' => false,
+        'full' => true,
+    ],
+    'issue-2370-A (lite)' => [
```

Comment:
Please rename this test to "issue-2475-A (lite)" and move this test above "issue-2475-B"

Reply:
done